BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -47,7 +47,10 @@ class ToolbarButton: UIButton, ThemeApplicable {
         configureLongPressGestureRecognizerIfNeeded(for: element)
         shouldDisplayAsHighlighted = element.shouldDisplayAsHighlighted
 
-        let image = UIImage(named: element.iconName)?.withRenderingMode(.alwaysTemplate)
+        let image = UIImage(named: element.iconName)?
+            .withRenderingMode(.alwaysTemplate)
+            .imageFlippedForRightToLeftLayoutDirection()
```

Comment:
Not all the buttons should be flipped - only back, forward and the cancel edit arrow. This is why the ticket asks for a config so we can set this individually.